### PR TITLE
vowls-index solution and test

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: tests/tolower_test/main.go [hunk 1]
```diff
@@ -1,17 +1 @@
...
-	"github.com/01-edu/go-tests/lib/random"
-	"github.com/01-edu/go-tests/solutions"
-)
-
-func main() {
-	table := append(random.StrSlice(chars.ASCII), "Hello! How are you?")
-	for _, arg := range table {
-		challenge.Function("ToLower", student.ToLower, solutions.ToLower, arg)
-	}
-}
```
Comment (davidrobert99) [inline tests/tolower_test/main.go:17]: Why was this deleted?

File: tests/vowelsindex_test/main.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+	table := []string{"student", "hello Iyan", "world", "Hello! How are you?"}
+
```
Comment (davidrobert99) [inline tests/vowelsindex_test/main.go:12]: Probably add more test cases like for example "AAEO$%&UOTX" or "*+-aeiouAEMNOoa"

**Review by Hamzaelkhatri — APPROVED** [review]
LGTM , just fix format of `solutions/vowelsindex.go`

File: solutions/vowelsindex.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package solutions
+
+
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply solutions/vowelsindex.go:3]: done

File: tests/vowelsindex_test/main.go [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+	table := []string{"student", "hello Iyan", "world", "Hello! How are you?", "AAEO$%&UOTX", "AEMNOoa*+-aeiou", "wo5opi=ws8;eza"}
```
Comment (MSilva95) [inline tests/vowelsindex_test/main.go:11]: add tests that don't have vowels and empty tests

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: tests/vowelsindex_test/main.go (hunk 4 above)
Comment (MSilva95) [reply tests/vowelsindex_test/main.go]: Add the test 

{"ptghjkvl"}

**Review by MSilva95 — APPROVED** [review]
